app/api/events/upcoming/route.ts

```diff
@@ -0,0 +1,16 @@
+import { hyphenateUrl } from "app/api/hyphenateUrl";
+import { NextRequest } from "next/server";
+import client from "../../../../tina/__generated__/client";
+import { getEventsWithClient } from "../getEvents";
+
+export async function GET(req: NextRequest) {
+  let presenterName = req.nextUrl.searchParams.get("presenterName");
+  if (!presenterName)
+    return new Response("presenterName is required", { status: 400 });
```

Comment:
```suggestion
  if (!presenterName) {
    return new Response("presenterName is required", { status: 400 });
  }
  
```
braces for readability

Reply:
I've removed the 400 since this is no longer relevant (See comment https://github.com/SSWConsulting/SSW.Website/pull/2828#issuecomment-2232047081)


However I did update he if statements in this function to use braces.